storage/storage_dest.go

```diff
@@ -239,7 +239,7 @@ func (s *storageImageDestination) putBlobToPendingFile(stream io.Reader, blobinf
 	filename := s.computeNextBlobCacheFile()
 	file, err := os.OpenFile(filename, os.O_CREATE|os.O_TRUNC|os.O_WRONLY|os.O_EXCL, 0600)
 	if err != nil {
-		return private.UploadedBlob{}, fmt.Errorf("creating temporary file %q: %w", filename, err)
+		return private.UploadedBlob{}, err
```

Comment:
I think it’s somewhat valuable to explain why the user is suddenly seeing a path what wasn’t in the input.